src/command/cmd_funcs.c

```diff
@@ -7258,14 +7258,25 @@ cmd_plugins(ProfWin* window, const char* const command, gchar** args)
         }
     }
 
+    GSList* unloaded_plugins = plugins_unloaded_list();
```

Comment:
`/plugins` should list all loaded plugins and also all the plugins that are available on the global installation path.
When did you get into the situation that you need to print the plugins like you did here?
When they are in `.local/share/profanity/plugins` but unloaded by hand?

Reply:
Yes. All plugins can be loaded only locally (from `$XDG_DATA_HOME/profanity/plugins/`) , as far as I understood. From other paths they must be installed to `$XDG_DATA_HOME/profanity/plugins/`. The only difference between determined global path and custom path - the ability to skip the full path when installing:
```shell
/plugins install /tmp/profanity-plugins/stable/say.py  # install  and load locally
/plugins install say.py   # same as /plugins install GLOBAL_PYTHON_PLUGINS_PATH/say.py
/plugins install pid.so   # same as  /plugins install GLOBAL_C_PLUGINS_PATH/pid.so
```

Current message is misleading:
https://github.com/profanity-im/profanity/blob/fcc20628d5a01c3d1c7a26fca48d79631eaac93d/src/command/cmd_funcs.c#L7267
Prints only loaded plugins. Getting information about unloaded (but still installed) plugins could also be useful.